compliance_checker/imos/util.py

```diff
@@ -194,8 +194,7 @@ def find_quality_control_variables(dataset):
                 quality_control_variables.append(var)
                 continue
 
-        if getattr(var, 'flag_values', None) is not None or getattr(var,\
-                    'flag_meanings', None) is not None:
+        if getattr(var, 'quality_control_conventions', None) is not None:
```

Comment:
If we don't need to do anything with the value of the attribute, it's simpler to say
``` python
if hasattr(var, 'quality_control_conventions'):
```